CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
 - Added code logic to send `VIRTIO_VSOCK_EVENT_TRANSPORT_RESET` on snapshot
   creation, when the Vsock device is active. The event will close active
   connections on the guest.
+- Added `GET` request on `/vm/config` that provides full microVM configuration
```

Comment:
Should we also add an example in the docs ?

Reply:
I don't see any good place for this, also it's a simple GET with no body, not sure what the example would achieve.

Reply:
I agree that the example would be trivial.